openforcefield/tests/test_molecule.py

```diff
@@ -372,6 +372,59 @@ def test_to_networkx(self, molecule):
         """Test conversion to NetworkX graph."""
         graph = molecule.to_networkx()
 
+    @pytest.mark.parametrize('molecule', mini_drug_bank())
+    def test_to_inchi(self, molecule):
+        """Test conversion to standard and non-standard InChI"""
+        inchi = molecule.to_inchi()
+        non_standard = molecule.to_inchi(True)
+
+    @pytest.mark.parametrize('molecule', mini_drug_bank())
+    def test_to_inchikey(self, molecule):
+        """Test the conversion to standard and non-standard InChIKey"""
+        inchikey = molecule.to_inchikey()
+        non_standard_key = molecule.to_inchikey(True)
+
+    def test_from_bad_inchi(self):
```

Comment:
Could you move this test to `test_toolkits` and parametrize it to run separately on RDKit and Openeye?

Reply:
Good point ill make sure these are all moved to cover RDKit as well.